meetings/2021-11-17.md

```diff
@@ -0,0 +1,122 @@
+    names
+  * Right now 3 browser, development, production
+  * Added some guidance as to when conditions can be added to the list
+    * use case for why should be added to the list
+    * should be some existing implementations
+    * etc.
+  * PR adds 2 new ones, types and deno. Types was added by angular. They do have
+    others that we have not listed, but did not want to open that can of worms yet.
+  * Could also be something that we move to the a different repo, place
+  * Matteo a few concerns, but resolved in the current text
+    * old text seemed to imply we supported them
+    * branding/marketing
+  * Michael - would this make more sense under the OpenJS Foundation
+  * Guy, a while back considered a package.json spec, but pushback on that.
+  * Matteo fine with it staying Node.js project
+  * Michael Z, ok with it staying, but one advantage with being outside, is that it does not need to
+    be linked to the Node.js documentation.
+  * Antoine +1 to what Michael Z said
+  * Michael point that resonated was documenting something that project has not documented.
+  * Michael Z, also applies to types one, would be good to know that the project wants/intends to define
+  * Guy, if getting confirmation of project interest make sense to people could be good criteria
+  * Ronag, kinds of feels out of scope for Node.js project
+    * which ones are relevant to Node.js
+    * Matteo in strict terms, maybe just types
+  * Development/production in webpack
+  * No objection to landing in core if we get implementor feedback confirmation,
+    move out later if there is enough momentum to do that later.
+
+* Rename default branch from "master" to "main" [#33864](https://github.com/nodejs/node/issues/33864)
+  * No updates this week.
+
+* Migration of core modules to primordials [#30697](https://github.com/nodejs/node/issues/30697)
+  * <https://github.com/nodejs/TSC/issues/1104> opened to discuss possible options
+  * Ruben has listed a number of additional options that we should consider
+  * Ronag, having vote on those 3 is not feasible
+  * Trott, question of having vote on 3 is different from if those 3 is right.
+    * too many will not result in positive outcome
+  * Possible goals
+    * Tamper Free everywhere
+    * Tamper Free just for module loading
+    * Selectively if possible without negative performance
+    * Not tamper free
+  * Approaches
+    * primordials
+    * move required code to C++
+  * Ronag from first question, only viable option is Tamper Free just for X
+  * Rich probably want to use tamper-resistant
+  * Gireesh, performance should not be a consideration if people looking for hyper
+    Performant for areas which are security critical. Rich not sure everybody will agree with that.
+  * Gireesh, don’t use current performance as baseline.
+  * Rich some people advocate for -> correctness is what matter most. But in terms of traction
+     on the TSC the position on perf/correctness has not traction.
```

Comment:
I'm not sure what that I had said this is trying to summarize. I'd be OK deleting it unless it's important somehow to the summary?

```suggestion
```